src/components/ButtonPanel.js

```diff
@@ -20,6 +21,8 @@ export default class ButtonPanel extends Component {
     let buttonsView = [];
 
     const { buttonGroups } = this.state;
+    const { clickHandler } = this.props;
+
```

Comment:
* [ ] you should implement `handleClick` function, then pass that to the `Button` component ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone5/panel/implementHandleClick.png))

```javascript
handleClick = (buttonName) => {
  this.props.clickHandler(buttonName);
};
```

Reply:
Done.